frontend/README.md

```diff
@@ -0,0 +1,39 @@
+# frontend
+
+This template should help get you started developing with Vue 3 in Vite.
```

Comment:
Seems unrelated to your project. It's not a template anymore. I would clean up this readme.